src/lib/styles/themes/dark.scss

```diff
@@ -63,6 +63,10 @@
   --disable: #505050;
   --disable-contrast: #969696;
 
+  // Line
+  --line: #505050;
```

Comment:
Out of curiosity, where are we gonna use it?

Reply:
Mischa said "there are lots of lines in the design" and also that he use the term "line" in his Figma file.

Currently we use the line in the neuron of the proposal detail and we will use also the color as background of the detail (see screenshot in red)
<img width="1510" alt="Capture d’écran 2022-09-02 à 12 45 49" src="https://user-images.githubusercontent.com/16886711/188126033-e3f596f5-4796-439f-af79-e970411d3268.png">

Reply:
Thank you for the update.